src/mnemonic.rs

```diff
@@ -195,10 +198,8 @@ mod test {
             let m = values[1].as_str().unwrap();
             let mnemonic = Mnemonic::from_str(m).unwrap();
             let seed = mnemonic.to_seed(Some("TREZOR"));
-            assert_eq!(
-                mnemonic.to_string(),
-                Mnemonic::new(data.as_slice()).unwrap().to_string()
-            );
+            // Test Display for Mnemonic
+            assert_eq!(format!("{}", mnemonic), Mnemonic::new(data.as_slice()).unwrap().to_string());
```

Comment:
```suggestion
            assert_eq!(format!("{}", mnemonic), Mnemonic::new(data.as_slice()).unwrap());
```
Maybe test `Display` directly?

Reply:
Let's do that!

Reply:
@jeandudey it seems that the `format!`-like macros is the only way to test `Display` implementation without boilerplate.